app/app.js

```diff
@@ -1,10 +1,12 @@
 const express = require("express");
-const { getAllTopics } = require("./controllers");
+const { getAllTopics, getAllEndpoints } = require("./controllers");
 const { handlePsqlErrors, handleCustomErrors, handleServerErrors } = require("./errors");
 
 const app = express();
+app.use(express.json())
```

Comment:
This line isn't currently needed and should be removed, as it helps you access the body of a post/patch request - so isn't relevant to this ticket 🎫